verification-http-client/Cargo.toml

```diff
@@ -9,15 +9,15 @@ license-file = "../LICENSE"
 base64 = "0.9"
 bytes = "0.4"
 crossbeam = "0.3"
-flate2 = "1.0"
+flate2 = "1.0.24"
```

Comment:
This bumps us to a version which has a native miniz implementation rather than the C implementation which fails to link.